src/items/itemsTable.component.tsx

```diff
@@ -0,0 +1,282 @@
+      },
+      {
+        header: 'Usage Status',
+        accessorFn: (row) => {
+          // Assuming row.usage_status contains the numeric value corresponding to the enum
+          const status = Object.values(UsageStatusType).find(
+            (value) =>
+              UsageStatusType[value as keyof typeof UsageStatusType] ===
+              row.usage_status
+          );
+          return status || 'Unknown';
+        },
+        size: 200,
+        filterVariant: 'select',
+      },
+      {
+        header: 'Notes',
+        accessorFn: (row) => row.notes ?? '',
+        size: 250,
+        Cell: ({ row }) =>
+          row.original.notes && (
+            <Tooltip
+              title={row.original.notes}
+              placement="top"
+              enterTouchDelay={0}
+              arrow
+              aria-label={`Catalogue item description: ${row.original.notes}`}
+            >
+              <InfoOutlinedIcon />
+            </Tooltip>
+          ),
+      },
+      ...viewCatalogueItemProperties.map((property, index) => ({
+        header: `${property.name} ${property.unit ? `(${property.unit})` : ''}`,
+        accessorFn: (row: Item) => {
+          if (property.type === 'boolean') {
+            return (findPropertyValue(
+              row.properties,
+              property.name
+            ) as boolean) === true
+              ? 'Yes'
+              : 'No';
+          } else if (property.type === 'number') {
+            return typeof findPropertyValue(row.properties, property.name) ===
+              'number'
+              ? findPropertyValue(row.properties, property.name)
+              : 0;
+          } else {
+            // if the value doesn't exist it return type "true" we need to change this
+            // to '' to allow this column to be filterable
+
+            return findPropertyValue(row.properties, property.name);
+          }
+        },
+        size: 250,
+        filterVariant:
+          propertyFilters[
+            property.type as 'string' | 'boolean' | 'number' | 'null'
+          ],
+
+        Cell: ({ row }: { row: MRT_Row<Item> }) => {
+          if (
+            typeof findPropertyValue(row.original.properties, property.name) ===
+            'number'
+          ) {
+            return findPropertyValue(row.original.properties, property.name) ===
+              0
+              ? 0
+              : findPropertyValue(row.original.properties, property.name) !==
+                null
+              ? findPropertyValue(row.original.properties, property.name)
+              : '';
+          } else if (
+            typeof findPropertyValue(row.original.properties, property.name) ===
+            'boolean'
+          ) {
+            return findPropertyValue(row.original.properties, property.name)
+              ? 'Yes'
+              : 'No';
+          } else {
+            return findPropertyValue(row.original.properties, property.name);
+          }
+        },
+      })),
```

Comment:
Is this completely the same as the items table - if so could it be put in a function and exported instead? I think it would even resolve the missing coverage as it would definitely be used then.

Reply:
That's is going to be done after MVP